test/Form_test.js

```diff
@@ -70,14 +69,29 @@ describe("Form", () => {
     const formData = {foo: "invalid"};
 
     function FieldTemplate(props) {
-      const {id, classNames, label, help, required, description, errors, children} = props;
+      const {
+        id,
+        classNames,
+        label,
+        help,
+        rawHelp,
+        required,
+        description,
+        rawDescription,
+        errors,
+        rawErrors,
+        children,
+      } = props;
       return (
         <div className={"my-template " + classNames}>
           <label htmlFor={id}>{label}{required ? "*" : null}</label>
           {description}
+          <span id={`${id}__raw_description`}>{`${rawDescription} rendered from the raw format`}</span>
           {children}
           {errors}
+          {rawErrors ? <ul>{rawErrors.map((error, i) => <li key={i} className="raw-error">{error}</li>)}</ul> : null}
```

Comment:
Nit: Would you mind spreading this line onto multiple ones to improve legibility?